src/index.ts

```diff
@@ -5,6 +5,11 @@ import open from 'open';
 import sendNotification from './notification';
 import {Logger} from './logger';
 
+/**
+ * Send test email
+ */
+sendNotification('test');
+
```

Comment:
Let's leave this one out. #24 covers this, but I'd like to see it do a little more.

```suggestion
```